### ISSUE TITLE
FPS Boost Unable to Find LoL Client Exe

### ISSUE THREAD

**Issue by morbidbug** [DjordjeMandic/LeagueFPSBoost#6]
Hi i recently downloaded your League of Legends FPS boost so I can stop dealing with fps spikes when playing on my low-end pc. I downloaded the LeagueFPSBoost exe file and put it on my desktop. Once I double-clicked, a window came up to select the League of Legends Client exe file but when I did, an error window popped up saying "Error while setting custom Lol path: BsSndRpt.exe or LeagueClient.exe missing. Maybe wrong file selected?". I tried uninstalling and reinstalling league but it still did not work. I know your fps boost is like 2 years old, is that why? Thank you!

**Comment by DjordjeMandic** [comment]
Yeah, client has changed. I based the algorithm for path finding to rely on few files that client used to have. Since they merged usernames and added the launcher for client current algorithm cant find the path. I can try to fix it if i get some free time but i doubt that will be only problem. I didnt play league for long time so i stopped updating this. Sadly there is no fix that can be applied from your side. Only way to fix it is to do a code change and recompile everything. TBH i forgot how i even implemented the updater and etc xd... Its going to take a while to read all that code again. But if i get some free time i will look into it. Thanks for mentioning this to me

**Comment by morbidbug** [comment]
Yes of course :3 thought it was worth a try haha no worries. Do you happen to know of any other fps boosts downloaders or ways to get a potato to run league smoothly? I tested my laptop and it says I have the requirements to play League, the fps just spikes so bad. Thank you anyway for your response :)

**Comment by DjordjeMandic** [comment]
@morbidbug you can try to manually set league game client to high priority and actual league client to below normal priority in task manager. That's what my program does and that's how it gains most fps. League game client is process that gets launched after champion select and it goes to high-priority. At same time you should set leagueclient priority to below normal.
